spec/factories/files.rb

```diff
@@ -52,6 +52,13 @@
       role { 'transcription' }
     end
 
+    trait :model do
```

Comment:
to consider:  make this model_3d as well?   Model isn't enough for me to recognize it as 3d.

Reply:
makes sense. updated